instrumentation/compile-stub/README.md

```diff
@@ -0,0 +1,4 @@
+# Compile stubs
+In case instrumentation code needs a class that is not available in a public repository you can add a stub class that has the methods called from integration code to this module.
```

Comment:
```suggestion
In case instrumentation code needs a class from the instrumented library that is not available in a public repository you can add a stub class that has the methods called from integration code to this module.
```